source/geh_common/src/geh_common/testing/spark/spark_test_session.py

```diff
@@ -88,6 +94,10 @@ def _make_default_config(data_dir: Path) -> dict:
     Returns:
         A dictionary of default configuration values.
     """
+    warehouse_path = f"{data_dir.resolve()}/__spark-warehouse__"
+    temp_path = f"{data_dir.resolve()}/__spark_tmp__"
+    metastore_path = f"{data_dir.resolve()}/__metastore_db__"
```

Comment:
The `__` originated from the files originally appearing in the source tree. So we could easily detect and ignore them.
```suggestion
    warehouse_path = f"{data_dir.resolve()}/spark_warehouse"
    temp_path = f"{data_dir.resolve()}/spark_tmp"
    metastore_path = f"{data_dir.resolve()}/metastore_db"
```